### ISSUE TITLE
token.expires_on field is null on Signup

### ISSUE THREAD

**Issue by AMilkov** [nkostadinov/yii2-user#3]
```
Integrity constraint violation – yii\db\IntegrityException

SQLSTATE[23502]: Not null violation: 7 ERROR: null value in column "expires_on" violates not-null constraint
The SQL being executed was: INSERT INTO "token" ("type", "user_id", "created_at", "code") VALUES (0, 10, 1460025601, '_Bb_31JFpDcj0BIX_4PvvAHyW-7BLpQT')

in ...\vendor\yiisoft\yii2\db\Command.php at line 809
1. in ...\vendor\yiisoft\yii2\db\Schema.php at line 601
2. in ...\vendor\yiisoft\yii2\db\Command.php at line 824 – yii\db\Schema::convertException(PDOException, 'INSERT INTO "token" ("type", "us...')
3. in ...\vendor\yiisoft\yii2\db\Command.php at line 369 – yii\db\Command::queryInternal('fetch', null)
4. in ...\vendor\yiisoft\yii2\db\pgsql\Schema.php at line 453 – yii\db\Command::queryOne()
5. in ...\vendor\yiisoft\yii2\db\ActiveRecord.php at line 444 – yii\db\pgsql\Schema::insert('token', ['type' => 0, 'user_id' => 10, 'created_at' => 1460025601, 'code' => '_Bb_31JFpDcj0BIX_4PvvAHyW-7BLpQT'])
6. in ...\vendor\yiisoft\yii2\db\ActiveRecord.php at line 415 – yii\db\ActiveRecord::insertInternal(null)
7. in ...\vendor\yiisoft\yii2\db\BaseActiveRecord.php at line 575 – yii\db\ActiveRecord::insert(false, null)
8. in ...\vendor\yiisoft\yii2\db\BaseActiveRecord.php at line 1396 – yii\db\BaseActiveRecord::save(false)
9. in ...\vendor\yiisoft\yii2\db\BaseActiveRecord.php at line 1185 – yii\db\BaseActiveRecord::bindModels(['user_id' => 'id'], nkostadinov\user\models\Token, nkostadinov\user\models\User)
10. in ...\vendor\nkostadinov\yii2-user\components\User.php at line 128 – yii\db\BaseActiveRecord::link('user', nkostadinov\user\models\User)
11. in ...\vendor\nkostadinov\yii2-user\models\forms\SignupForm.php at line 79 – nkostadinov\user\components\User::register(nkostadinov\user\models\User)
12. in ...\vendor\nkostadinov\yii2-user\controllers\RegistrationController.php at line 62 – nkostadinov\user\models\forms\SignupForm::signup()
13. nkostadinov\user\controllers\RegistrationController::actionSignup()
14. in ...\vendor\yiisoft\yii2\base\InlineAction.php at line 53 – call_user_func_array([nkostadinov\user\controllers\RegistrationController, 'actionSignup'], [])
15. in ...\vendor\yiisoft\yii2\base\Controller.php at line 157 – yii\base\InlineAction::runWithParams([])
16. in ...\vendor\yiisoft\yii2\base\Module.php at line 445 – yii\base\Controller::runAction('signup', [])
17. in ...\vendor\yiisoft\yii2\web\Application.php at line 84 – yii\base\Module::runAction('user/registration/signup', [])
18. in ...\vendor\yiisoft\yii2\base\Application.php at line 389 – yii\web\Application::handleRequest(yii\web\Request)
19. in ...\web\index.php at line 12 – yii\base\Application::run()
```

**Comment by nkostadinov** [comment]
@AMilkov I see you are using pgsql and may be Yii AR handles empty values differently because there are no problems in mysql. I made a new migration setting default value for field expires_on and this should fix this issue. Please update and let me know if there are other problems using this extension.

**Comment by AMilkov** [comment]
Niki, you are right - PostgreSQL is more strict and assign NULL to the fields that are not set. 
The fix is working OK. 
However in order for migration to work for PostgreSQL it has to be slightly modified, here is the PR:
https://github.com/nkostadinov/yii2-user/pull/4

Otherwise the migrations bug is well known of the YII team and is scheduled to be fixed in 2.1.0 
https://github.com/yiisoft/yii2/issues/9903

**Comment by AMilkov** [comment]
Niki, you are right - PostgreSQL is more strict and assign NULL to the fields that are not set. 
The fix is working OK. 
However in order for migration to work for PostgreSQL it has to be slightly modified, here is the PR:
https://github.com/nkostadinov/yii2-user/pull/4

Otherwise the migrations bug is well known of the YII team and is scheduled to be fixed in 2.1.0 
https://github.com/yiisoft/yii2/issues/9903
